src/components/SMARTBox/PatientBox.js

```diff
@@ -245,12 +244,7 @@ export default class SMARTBox extends Component {
         flat: true,
       })
       .then((result) => {
-        this.setState((prevState) => ({
-          serviceRequests: {
-            ...prevState.serviceRequests,
-            [patientId]: result,
-          },
-        }));
+        this.setState((prevState) => ({ serviceRequests: result }));
```

Comment:
We don't need to pass prevState as we are not using it.

Reply:
Is this something that should be removed with a future PR?

Reply:
It should be an easy change. Just remove the parameter :-)